ynr/apps/people/migrations/0044_populate_biography_timestamp.py

```diff
@@ -0,0 +1,33 @@
+from candidates.diffs import get_version_diffs
+from django.db import migrations
+
+
+def populate_biography_last_updated_timestamp(apps, schema_editor):
+    Person = apps.get_model("people", "Person")
+    for person in Person.objects.all():
+        try:
+            diffs = get_version_diffs(person.versions)
+            if diffs:
+                for diff in diffs:
+                    if diff["data"]["biography"]:
```

Comment:
I get loads of key errors on biography when I run this. If you changed this line to `diff["data"].get("biography")` most of them would go away I think

Reply:
Actually, I think there's another change:

If there's not biography in the latest version (or set on the person model), then we don't need to check the previous versions: there's no biography.

I think we can wrap this in a `if person.biography` line, and then loop through versions to find the most recent edit to it. 

At the moment we're looping through all versions of all people even if they don't have a current statement. 

You could speed this up more by filtering the QS on `.exclude(biography="")` too.